presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveBucketedTables.java

```diff
@@ -64,16 +64,16 @@ public Requirement getRequirements(Configuration configuration)
     public void testIgnorePartitionBucketingIfNotBucketed()
     {
         String tableName = mutableTablesState().get(BUCKETED_PARTITIONED_NATION).getNameInDatabase();
-        populateDataToHiveTable(tableName, NATION.getName(), "part_key = 'insert_1'");
```

Comment:
> Rename method

This belongs together with `Remove redundant Optional` commit ;)

See: by removing optional-ness, you made this method specific to partitioned table, hence my suggestion to change the name.

Reply:
Oh... thanks for explanation. Fixed.